.gitignore

```diff
@@ -20,3 +21,4 @@ node_modules
 deploy_logs.json
 
 temp
+Makefile
```

Comment:
why did you add it to gitignore?

Reply:
![image](https://github.com/user-attachments/assets/01fb6d9b-3fd9-4d70-a87c-5820e2fb939e)
Because I had a problem with push.